src/SortedSet/BinaryTreeStrategy.coffee

```diff
@@ -35,19 +36,25 @@ binaryTreeDelete = (node, value, comparator) ->
 
 class BinaryTreeStrategy extends AbstractBinaryTreeStrategy
   constructor: (@options) ->
+    super()
     @comparator = @options.comparator
+    @onInsertConflict = @options.onInsertConflict
     @root = null
 
   insert: (value) ->
     compare = @comparator
+    onInsertConflict = @onInsertConflict
```

Comment:
Nix this line and use `@onInsertConflict` below.

`compare = @comparator` is here to speed up the common code path. This `onInsertConflict = @onInsertConflict` doesn't accomplish that.